pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -100,10 +102,12 @@ func (r *ReconcileStorageCluster) newStorageClasses(initData *ocsv1.StorageClust
 				"imageFeatures":             "layering",
 				"csi.storage.k8s.io/fstype": "ext4",
 				"imageFormat":               "2",
-				"csi.storage.k8s.io/provisioner-secret-name":      "rook-csi-rbd-provisioner",
-				"csi.storage.k8s.io/provisioner-secret-namespace": initData.Namespace,
-				"csi.storage.k8s.io/node-stage-secret-name":       "rook-csi-rbd-node",
-				"csi.storage.k8s.io/node-stage-secret-namespace":  initData.Namespace,
+				"csi.storage.k8s.io/provisioner-secret-name":            "rook-csi-rbd-provisioner",
```

Comment:
@humblec in current storageclass `AllowExpansion` is set to false, are you planning to change it to true?

Reply:
As mentioned here https://github.com/openshift/ocs-operator/pull/488#issuecomment-621108180 depends on some decisions we can make it `true`, waiting for the same.

Reply:
Enabling allowVolumeExpansion .